client/app/main/main.component.js

```diff
@@ -65,7 +65,7 @@ export class MainController {
       leanplum.unregisterFromWebPush();
     } else {
       console.log('Webpush subscribing user...');
-      leanplum.registerForWebPush('/sw.min.js', subscriptionStatus => {
+      leanplum.registerForWebPush('/myServiceWorker.js', subscriptionStatus => {
```

Comment:
Thanks for the PR. This looks redundant to me. Isn't it better to keep it as sw.min.js since this file is generated from grunt? Or else we need to rename it every time.

Reply:
That is correct. If we want to rename it, please also adjust the output config in grunt.

Reply:
I didn't realize we had something doing that. I think the correct change for this sample would be to completely remove the generation parts. The point of this change is that people could use their own service worker. That's why I changed the name. They would want to use their own checked in service worker, not have a script to copy it from Leanplum. In general, I think this sample project has way too many moving parts for a "hello world" example for using our web push API. If you agree I will remove gulp tasks "copy:sw:*".